### PR TITLE
SVCPLAN-5555: Add SUSE support

### PR DESCRIPTION
**Pull request by billglick**
This is tested on `control-test-opensuse155a`.

```
Add    puppet/zypprep module
Update ncsa/pam_access                  to tag ncsa/1.1.1 BREAKING change
    pam_access::pam::redhat renamed to pam_access::pam::authselect
Update ncsa/profile_acctd               to tag v1.2.2
Update ncsa/profile_additional_packages to tag v0.4.2
Update ncsa/profile_audit               to tag v0.1.18
Update ncsa/profile_backup              to tag v0.0.10
Update ncsa/profile_duo                 to tag v1.0.6
Update ncsa/profile_email               to tag v0.2.7
Update ncsa/profile_firewall            to tag v1.1.3
Update ncsa/profile_hardening           to tag v0.4.1
Update ncsa/profile_java                to tag v0.1.1
Update ncsa/profile_lvm                 to tag v1.0.1
Update ncsa/profile_monitoring          to tag v0.3.3
Update ncsa/profile_motd                to tag v0.3.3
Update ncsa/profile_nfs_client          to tag v0.2.1
Update ncsa/profile_pam_access          to tag v0.0.7
Update ncsa/profile_puppet_agent        to tag v0.2.0 BREAKING change
    param yumrepo renamed to repos
Update ncsa/profile_rsyslog             to tag v0.3.1
Update ncsa/profile_selinux             to tag v0.1.1
Update ncsa/profile_session_log         to tag v1.0.1
Update ncsa/profile_system_auth         to tag v0.4.1
Update ncsa/profile_update_os           to tag v0.4.4
Update ncsa/profile_virtual             to tag v0.1.2
Update ncsa/sshd                        to tag v0.5.3
Update ncsa/sssd                        to tag v3.0.3
Update ncsa/telegraf                    to tag ncsa/v0.1.2
Change sssd::domains for illinois.edu to use os default ldap_tls_cacert from ncsa/sssd module
```

### CODE REVIEW

File: data/common.yaml [hunk 1]
```diff
@@ -246,7 +246,6 @@ sssd::domains:
     ldap_group_search_base: "ou=groups,dc=ncsa,dc=illinois,dc=edu"
     ldap_schema: "rfc2307bis"
     ldap_search_base: "dc=ncsa,dc=illinois,dc=edu"
-    ldap_tls_cacert: "/etc/pki/tls/certs/ca-bundle.crt"
```
Comment (bsper2) [inline data/common.yaml:249]: Why is this getting removed?

Is it something we don't want/need in SUSE but still need in the Redhat family?

Comment (billglick) [reply data/common.yaml:249]: If we remove it, the `ncsa/sssd` module uses the default value that comes from the `data/os/...` in the module, which matches what we want for both RedHat and SUSE. The value we were already expecting gets put in place for RHEL and the different path for SUSE comes in as expected as well.

**Review by bsper2 — APPROVED** [review]
Looks good. Thanks for including a quick summary of what the breaking changes are in the commit message.

File: data/common.yaml (hunk 1 above)
Comment (bsper2) [reply data/common.yaml:249]: Ah yup I see

**Pull request merged**